Client/Frontend/Browser/Tab.swift

```diff
@@ -263,6 +263,8 @@ class Tab: NSObject {
     }
     
     func deleteWebView() {
+        contentScriptManager.uninstall(from: self)
```

Comment:
Do we need to do this anywhere else? Should this be placed inside the `if webView = webView` condition?

Reply:
From my tests this is the only place where contents scripts are leaking, 

I can move it under webview unwrapping, but we uninstall it with optional chaining anyway